donut/modules/arcfeedback/templates/complaint.html

```diff
@@ -0,0 +1,146 @@
+{% extends "layout.html" %}
+{% block page %}
+<div class="container theme-showcase" role="main">
+    <div class="jumbotron">
+	<div class="row">
+	    <div class="col-md-8">
+		<div class="panel panel-default">
+		    <div class="panel-heading">
+			<h3>Feedback Summary</h3>
+		    </div>
+		    <div class="panel-body" id="summary-panel">
+			<p>Course: {{complaint['course']}} </p>
+			<p>Status: {{complaint['status']}} </p>
+			{% if not complaint['emails'] %}
+			    <p>No emails subscribed to this complaint </p>
+			{% else %}
+			    <p>Subscribed emails: </p>
+			    {% for email in complaint['emails'] %}
+			        <p>{{email}}</p>
+			    {% endfor %}
+			{% endif %}
+		    </div>
+		    <button class="btn btn-primary" id="markRead">Mark as read</button>
+		    <button class="btn btn-primary" id="markUnread">Mark as unread</button>
+		</div>
+	    </div>
+	    <div class="col-md-4">
+		<div class="panel panel-default">
+		    <div class="panel-heading">
+			<h4>Add an email</h4>
+		    </div>
+		    <div class="panel-body">
+			<form method="POST" id="add-email" action="{{url_for('arcfeedback.arcfeedback_add_email', id=complaint['uuid'])}}">
+			    <div class="form-group">
+				<p>Email (or comma separated list of emails): </p>
+				<input class="form-control" type="email" name="email" />
+			    </div>
+			    <input class="btn btn-primary" type="submit" value="Add" id="add-email-btn">
+			</form>
+		    </div>
+		    <div class="row">
+		    </div>
+		</div>
+	    </div>
+	    <div class="row">
+		<div class="col-md-8 col-md-offset-2">
+		    <div>
+			<table class="table table-bordered" id="messageList">
+			    <thead class="thead-default">
+				<tr> <th>Time Posted</th> <th>Name</th> <th>Message</th> </tr>
+			    </thead>
+			    {% for message in complaint['messages'] %}
+			    <tr> 
+				<th>{{message['time'].strftime('%b %d %Y %-I:%M%p')}}</th> 
+				<th>{{ message['poster'] }}</th> 
+				<td>{{ message['message'] }}</td>
+			    </tr>
+			    {% endfor %}
+			</table>
+		    </div>
+		    <div class="panel panel-default">
+			<div class="panel-heading">
+			    <p>Add a comment<p>
+			</div>
+			<div class="panel-body">
+			    <form method="POST" id="addMessage" action="/">
+				<div class="form-group">
+				    <p>Name: </p>
+				    <input class="form-control" type="text" name="poster" />
```

Comment:
there's already a character limit on serverside for the name. Just add it here on client side too